Resources/fsh-generated/resources/Practitioner-PractitionerWalterArzt.json

```diff
@@ -97,7 +97,7 @@
           {
             "code": "11001",
             "system": "urn:oid:1.2.276.0.76.5.514",
-            "display": "\tFA Allgemeinmedizin"
+            "display": "FA Allgemeinmedizin"
```

Comment:
Die JSON brauch wir nicht bearbeiten, weil diese aus dem FSH Source durch die CI generiert werden. Bitte in der passenden *.fsh file editieren.